### ISSUE TITLE
training loss result

### ISSUE THREAD

**Issue by fredlll** [Junshk/CinCGAN-pytorch#7]
Hi, what is your loss after training step 1? How can I know my loss is stable to begin training step 2? Is there a rough guideline? 

I used 10 GPUs and after 30 epochs now the loss is 

===> Epoch[28](990/998): Loss: idt 0.000000 0.037730 cyc 0.000000  0.005994 D 0.000000 0.464715, G: 0.000000 0.284303, psnr_hr: 16.187803, psnr_lr 7.150141

and it is not improving

**Comment by zahra-anvari** [comment]
Hi,
Which torch version are you using?
